### PR TITLE
Readability add summaries

### PR DESCRIPTION
**Pull request by Emilkens**
Increased code readability by adding summaries on classes, their constructors and methods.

### CODE REVIEW

File: WakeDevice/TargetEndpoint.cs [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 namespace WakeDevice
 {
+    /// <summary>
+    /// Represents target endpoint network data
```
Comment (thresiukiewicz) [inline WakeDevice/TargetEndpoint.cs:13]: *targets endpoint

File: WakeDevice/TargetEndpoint.cs [hunk 2]
```diff
@@ -101,6 +112,12 @@ public TargetEndpoint(string mac, string ip, string? mask, List<uint>? ports)
 
             AddressPorts = ports;
         }
+        
+        /// <summary>
+        /// Wakes target device performing Wake On LAN
```
Comment (thresiukiewicz) [inline WakeDevice/TargetEndpoint.cs]: device performing?
"Wakes device by performing" was probably what it was supposed to mean, optionally could be a little more specific (eg. by sending magic packet to each port) but this one's up to dev

**Review by thresiukiewicz — COMMENTED** [review]
2 comments left in targetEndpoint, otherwise lgtm

File: WakeDevice/TargetEndpoint.cs (hunk 1 above)
Comment (Emilkens) [reply WakeDevice/TargetEndpoint.cs:13]: You may have misunderstood the concept behind that. An endpoint is a target itself, however there may be several endpoints and "target endpoint" simply specifies one particular endpoint. The word "target" is actually not necessary here. I'd stick to what it is now.

File: WakeDevice/TargetEndpoint.cs (hunk 2 above)
Comment (Emilkens) [reply WakeDevice/TargetEndpoint.cs]: Fixed by [commit 0fce650a29b8489cd7f4195e49f6f84281429df4 ](https://github.com/Emilkens/WakeDevice/pull/2/commits/0fce650a29b8489cd7f4195e49f6f84281429df4)

**Review by thresiukiewicz — APPROVED** [review]
lgtm

**Pull request merged**